diff --git a/dist/node/chunks/config.js b/dist/node/chunks/config.js
index 2b39a20c497ee5c9f1cefdf1781df9955b50b97d..cdbbb14fa9c1fac248725958f24b0c9c7c218a5b 100644
--- a/dist/node/chunks/config.js
+++ b/dist/node/chunks/config.js
@@ -29428,7 +29428,7 @@ function manifestPlugin() {
 				}
 				return manifestChunk;
 			}
-			const entryCssReferenceIds = cssEntriesMap.get(this.environment);
+			const entryCssReferenceIds = cssEntriesMap.get(this.environment) ?? [];
 			const entryCssAssetFileNames = /* @__PURE__ */ new Map();
 			for (const [name, id] of entryCssReferenceIds) try {
 				const fileName = this.getFileName(id);
